department-of-veterans-affairs/vets-website
# Pension Multiple Page Response Fixes

## Summary 
Fix errors in the Multiple Page Response pattern on the Pension Benefits form (21P-527EZ). 

### Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/91288
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/91296

### Associated Vets-api PR
n/a

### How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pensions_multiple_page_response flipper to 'enabled' at http://localhost:3000/flipper/pension_multiple_page_response
3. Run `vets-website` and `vets-api`
4. Go to `http://localhost:3001/pension/apply-for-veteran-pension-form-21p-527ez/` in your browser

### How to verify
1. View the AC for each issue and follow instructions
3. Verify that the form is meeting the expected behavior for each issue

### What areas of the site does it impact?
Pension Benefits Application

## Screenshots
n/a

### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
